data/brands/shop/wine.json

```diff
@@ -8,6 +8,7 @@
       "displayName": "Jacques’ Wein-Depot",
       "id": "jacquesweindepot-f93b7c",
       "locationSet": {"include": ["at", "de"]},
+	  "matchNames": ["Jacques Wein-Depot", "Jacques Weindepot"],
```

Comment:
These are redundant too because they only differ in punctuation from the `name` and `brand` tag values.